package.py

```diff
@@ -0,0 +1,18 @@
+import spack.package
+
+class SpackExercise(CMakePackage):
+    homepage = "https://simulation-software-engineering.github.io/homepage/"
+    url = "https://github.com/Simulation-Software-Engineering/spack-exercise/archive/refs/tags/v0.3.0.tar.gz"
+
+    maintainers("M-Jafarkhani")
+
+    license("MIT-license", checked_by="M-Jafarkhani")
+
+    version("0.3.0", sha256="e54a4c037941d85a22fb3e6e73195df8448cf69a96aa44ef374ac518344812f0")
+    version("0.2.0", sha256="3dd6b4cc0f7aff179d8e290bc3879056237ae372738a4bd7222f6450fbcdfc77")
+    version("0.1.0", sha256="cac78e641cb703e3fe51956f91fe8347ac52f74ef037d8eadae5777c65a19a00")
+
+    depends_on("cxx", type="build")
+
+    depends_on("boost@1.65.1:+filesystem", when="@0.2.0:")
+    depends_on("yaml-cpp@0.7.0", when="@0.3.0:")
```

Comment:
```suggestion
    depends_on("yaml-cpp@0.7.0:", when="@0.3.0:")
```
We require *at least* version 0.7.0 of yaml-cpp.